lib/Service/MailManager.php

```diff
@@ -292,17 +291,17 @@ public function flagMessage(Account $account, string $mailbox, int $uid, string
 		}
 
 		// Only send system flags to the IMAP server as other flags might not be supported
-		if (in_array(strtolower("\\$flag"), self::ALLOWED_FLAGS, true)) {
-			try {
-				$normalized = '\\' . strtolower($flag);
+		$imapFlags = self::ALLOWED_FLAGS[$flag] ?? [];
```

Comment:
Are you sure `strtolower()` can be removed?
I see in the standard that it is `$Junk`, but Horde probably `strtolower()` already?

Reply:
yes, because that is what we get from the front-end and there we always use the lower case flags